fastfield_codecs/src/lib.rs

```diff
@@ -37,12 +37,15 @@ mod line;
 mod linear;
 mod monotonic_mapping;
 mod monotonic_mapping_u128;
+mod null_index;
 mod null_index_footer;
 
 mod column;
 mod gcd;
 mod serialize;
 
+pub use null_index::*;
```

Comment:
```suggestion
```

That's actually nicer to not expose symbol here. The module is perfectly nice.

Reply:
I added it to get temporarily rid of the annoying unused warnings